xgcm/test/test_grid.py

```diff
@@ -663,3 +659,26 @@ def test_multi_axis_input(all_datasets, func, periodic, boundary):
             serial = getattr(grid, func)(serial, axis, boundary=boundary_axis)
         full = getattr(grid, func)(ds[varname], axes, boundary=boundary)
         xr.testing.assert_allclose(serial, full)
+
+
+# @pytest.mark.parametrize("funcname", ["diff", "interp", "min", "max", "integrate", "average", "cumint"])
+# integrate, average and cumsum call xarray sum method directly and
+# not _wrap_and_replace_coords
+@pytest.mark.parametrize("funcname", ["diff", "interp", "min", "max"])
```

Comment:
I forgot `derivative`. Could you add it? See some more discussion about which functions to include in the main comments.